plugins/hls-tactics-plugin/src/Wingman/Types.hs

```diff
@@ -80,6 +80,16 @@ data Config = Config
   { cfg_feature_set          :: FeatureSet
   , cfg_max_use_ctor_actions :: Int
   , cfg_timeout_seconds      :: Int
+  , cfg_auto_gas             :: Int
```

Comment:
Out of curiosity: Why it's not "depth" instead of "gas"? I mean, even your description says that this is a kind of depth.

Reply:
I don't know; this is just what @TOTBWF and I have been calling it!